### DISCUSSION TITLE
Is there an issue with the basic example?

### DISCUSSION THREAD

**Discussion by moskito** [TarekRaafat/autoComplete.js#239] [Q&A] [answered]
Hello,
It seems that this example does not work ( from https://tarekraafat.github.io/autoComplete.js/#/usage?id=_2-script ):

```
...
<textarea id="autoComplete"></textarea>

<script type="text/javascript">
    // https://tarekraafat.github.io/autoComplete.js/#/usage?id=_2-script
    // API Basic Configuration Object
    const autoCompleteJS = new autoComplete({
        placeHolder: "Search for Food...",
        data: {
            src: ["Sauce - Thousand Island", "Wild Boar - Tenderloin", "Goat - Whole Cut"]
        },
        resultItem: {
            highlight: {
                render: true
            }
        }
    });
</script>
```

When you click on the selected suggestion, nothing happen (the list disappear but the text is not appended to the textarea).

Something is missing somewhere I guess. Or is this the basic way of working?

Regards

**Comment by folknor** [comment, marked as answer]
Yes, that example is "missing" (it's not really meant to be a complete example) listening to the selection event, like
```js
$("autoComplete").addEventListener("selection", ev => { ... })
```
In that handler you can do `$("autoComplete").value = ev.detail.selection.value` or equivalent.

**Reply by folknor** [reply]
> (it's not really meant to be a complete example)

Now that I look at it again, maybe I'm wrong about that. Should wait for the author to respond :-)

**Comment by TarekRaafat** [comment]
Hello @moskito,

As @folknor thankfully mentioned that that example `"(it's not really meant to be a complete example)"` and the code he shared is the correct piece of code you're looking for as well and it could be done in another way as shown below.

```js
// Config Options
events: {
  input: {
    selection: (event) => {
      const feedback = event.detail;
      // Prepare User's Selected Value
      const selection = feedback.selection.value[feedback.selection.key];
      // Replace Input value with the selected value
      autoCompleteJS.input.value = selection;
    },
  },
},
```

Please let me know if you have any questions, and regarding the complete code of the example, I will make sure to add it.

Have a nice day all! :)

**Comment by moskito** [comment]
Thanks to both of you.
Could this be added to the documentation (it's a bit disturbing when you're discovering autocomplete.js)?
I would personally appreciate some more details / examples about the trigger / query parameters :)

@TarekRaafat: thanks for all the time spent on this project :)
